templates/zerver/emails/demo_request.html

```diff
@@ -0,0 +1,21 @@
+{% extends "zerver/emails/email_base_messages.html" %}
+
+{% block content %}
+<b>Subject</b>: Demo request for {{ organization_name }}
+<br />
+<b>Full Name</b>: {{ full_name }}
+<br />
+<b>Email</b>: {{ email }}
+<br />
+<b>Role</b>: {{ role }}
+<br />
+<b>Organization Type</b>: {{ organization_type }}
+<br />
+<b>Organization Website</b>: {{ organization_website }}
+<br />
+<b>Expected User Count</b>: {{ expected_user_count }}
+<br />
+<b>Message</b>: {{ message }}
+<br />
```

Comment:
I think we can use regular Zulip capitalization (e.g., "Organization type") here?

Reply:
Oops! Fixed.